packages/store/gas-report.json

```diff
@@ -693,7 +693,7 @@
     "file": "test/StoreCoreGas.t.sol",
     "test": "testSetAndGetField",
     "name": "set static field (1 slot)",
-    "gasUsed": 33280
+    "gasUsed": 33581
```

Comment:
+300 in gas; some increase is expected, is there anything we can do to further optimize this or is this as good as it gets?

Reply:
I didn't intentionally spend time optimizing gas here, probably can do better when I get to StoreCore